src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const VERY_SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCAUNT = 50;
```

Comment:
There is a typo in the constant name 'LONG_TERM_DISCAUNT'. It should be 'LONG_TERM_DISCOUNT'. Correct spelling is crucial for maintaining clear and understandable code.